pkgs/development/libraries/zydis/default.nix

```diff
@@ -13,33 +13,25 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "zydis";
-  version = "4.0.0";
+  version = "4.1.0";
 
   src = fetchFromGitHub {
     owner = "zyantific";
     repo = "zydis";
     rev = "v${version}";
-    hash = "sha256-/no/8FNa5LlwhZMSMao4/cwZk6GlamLjqr+isbh6tEI=";
+    hash = "sha256-akusu0T7q5RX4KGtjRqqOFpW5i9Bd1L4RVZt8Rg3PJY=";
   };
 
   nativeBuildInputs = [ cmake ];
   buildInputs = [ zycore ];
   cmakeFlags = [
-    "-DZYAN_SYSTEM_ZYCORE=ON"
     "-DCMAKE_INSTALL_LIBDIR=lib"
     "-DCMAKE_INSTALL_INCLUDEDIR=include"
+    "-DZYDIS_BUILD_TESTS=ON"
   ];
 
   doCheck = true;
   nativeCheckInputs = [ python3 ];
-  checkPhase = ''
-    pushd ../tests
-    python3 ./regression.py test ../build/ZydisInfo
-    python3 ./regression_encoder.py \
-      ../build/Zydis{Fuzz{ReEncoding,Encoder},TestEncoderAbsolute}
-    popd
-  '';
-
```

Comment:
The custom test invocation here is no longer necessary: Zydis' tests are [now integrated into CMake](https://github.com/zyantific/zydis/commit/a56a39f15efe64a41013ec142c028e6f72476232) and Nix invokes them automatically:

```
zydis> /nix/store/jacf2kn4dfj99c3ywbvfyg6w53xvxsfl-cmake-3.27.8/bin/ctest --force-new-ctest-process
zydis> Test project /build/source/build
zydis>     Start 1: ZydisRegression
zydis>     Start 2: ZydisRegressionEncoder
zydis> 1/2 Test #2: ZydisRegressionEncoder ...........   Passed    0.10 sec
zydis> 2/2 Test #1: ZydisRegression ..................   Passed    0.13 sec
zydis> 100% tests passed, 0 tests failed out of 2
zydis> Total Test time (real) =   0.13 sec
```